docs/frontend-system/architecture/05-extension-overrides.md

```diff
@@ -10,38 +10,149 @@ description: Frontend extension overrides
 
-<!--
+const app = createApp({
+ features: [ apertureOverrides ],
+});
 
-To override an existing extension you need to provide an extension through an extension override that has the same ID as the existing extension. That is, all of kind, namespace and name must match the extension you want to override.
+export default app.createRoot().
+```
 
-This means that you typically need to provide an explicit namespace when overriding extensions from a plugin.
+If the plugin you want to change is internal to your company or you just want to replace one of the application's core extensions, you can decide to store the overrides code directly in the app package or extract them to a separate package.
 
- - Example of how to create an extension override for an existing extension from some plugin
+Note that it can still be a good idea to split your overrides out into separate packages in large projects. But it's up to you to decide how to group the extensions into extension overrides.
 
- -->
+## Override Plugin Extensions
+
+To override an extension that is provided by a plugin, you need to provide a new extension that has the same ID as the existing extension. That is, all kind, namespace, and name options must match the extension you want to replace. This means that you typically need to provide an explicit `namespace` when overriding extensions from a plugin.
+
+:::info
+We recommend that plugin developers share the extension IDs in their plugin documentation, but usually you can infer the ID by following the [naming patterns](./08-naming-patterns.md) documentation.
+:::
+
+### Example
+
+Imagine you have a plugin with the ID `'search'`, and the plugin provides a page extension that you want to fully override with your own custom component. To do so, you need to create your page extension with an explicit `namespace` option that matches that of the plugin that you want to override, in this case `'search'`. If the existing extension also has an explicit `name` you'd need to set the `name` of your override extension to the same value as well.
+
+```tsx title="packages/app/src/search.ts" showLineNumbers
+import { createPageExtension } from '@backstage/frontend-plugin-api';
+
+// Creating a custom search page extension
+const customSearchPage = createPageExtension({
+  namespace: 'search',
+  // Omitting name since it is the index plugin page
+  defaultPath: '/search',
+  loader: () => import('./SearchPage').then(m => m.<SearchPage/>),
+});
+
+export createExtensionOverrides({
+  extensions: [customSearchPage]
+});
+```
+
+Don't forget to configure your overrides in the `createApp` function:
+
+```tsx title="packages/app/src/App.tsx" {5} showLineNumbers
+import { createApp } from '@backstage/frontend-app-api';
+import searchOverrides from './search';
+
+const app = createApp({
+  features: [searchOverrides],
+});
+
+export default app.createRoot();
+```
+
+Now let's talk about the last override case, orphan extensions.
+
+## Create Orphan Extensions
```

Comment:
```suggestion
## Create Standalone Extensions
```

Thinking this is perhaps a better term? "orphan" I think is perhaps a bit too easy to confuse with detached extensions that aren't part of the app tree?